packages/react/src/table/TableRow.tsx

```diff
@@ -1,67 +1,70 @@
 import { PropsWithChildren } from 'react';
 import { boxPalette, tokens } from '../core';
+import { theme } from '../ag-branding';
 import { useTableContext } from './TableContext';
 
 export type TableRowProps = PropsWithChildren<{
-	/** Indicates the current selected state of the table row. */
-	selected?: boolean;
 	/** The row index of the table row. */
 	'aria-rowindex'?: number;
+	/** Style the row when one cell contains an error. */
+	hasRowError?: boolean;
```

Comment:
I don't think we need to have `Row` on a `TableRow` component. Historically, we haven't used "state" prop prefixes like "has" and "is". Should we just use `invalid` like a Field does?

Reply:
>Should we just use invalid like a Field does?

This sounds reasonable to me. I did consider it but steered away since `invalid` has native html meaning and this is just so far removed from something that offers an `invalid` attribute. But yeah, I think from an DS adoptability point-of-view,  `invalid` is probably the best option. Thanks, mate.

- [ ] Switch `hasRowError` for `invalid`